src/actions/SendConfirmationActions.js

```diff
@@ -143,6 +148,7 @@ export const updateMaxSpend = () => (dispatch: Dispatch, getState: GetState) =>
   const walletId = getSelectedWalletId(state)
   const edgeWallet = getWallet(state, walletId)
   const spendInfo = getSpendInfo(state)
+  const toggleCryptoOnTopState = state.ui.scenes.sendConfirmation.toggleCryptoOnTop
```

Comment:
Do the incrementing in the reducer versus here. The reducer already has access to the current counter and can simply increment it.